ramls/automated-patron-blocks.raml

```diff
@@ -11,27 +11,80 @@ types:
   user: !include user.json
   loanPolicy: !include loan-policy.json
   loan: !include loan.json
+  account: !include accountdata.json
   errors: !include raml-util/schemas/errors.schema
   automatedPatronBlocks: !include automated-patron-blocks.json
+  synchronizationRequest: !include synchronization-request.json
+  synchronizationResponse: !include synchronization-response.json
+  synchronizationJob: !include synchronization-job.json
 
 traits:
   validate: !include raml-util/traits/validation.raml
+  language: !include raml-util/traits/language.raml
 
-/automated-patron-blocks/{userId}:
-  get:
-    responses:
-      200:
+/automated-patron-blocks:
+  /{userId}:
+    get:
+      responses:
+        200:
+          body:
+            application/json:
+              schema: automatedPatronBlocks
+              example: !include examples/automated-patron-blocks.sample
+        400:
+          description: "Invalid user ID in request"
+          body:
+            text/plain:
+              example: "Bad request, e.g. invalid user UUID"
+        500:
+          description: "Internal server error, e.g. due to misconfiguration"
+          body:
+            text/plain:
+              example: "Internal server error, contact administrator"
+  /synchronization:
+    /request:
+      post:
         body:
           application/json:
-            schema: automatedPatronBlocks
-            example: !include examples/automated-patron-blocks.sample
-      400:
-        description: "Invalid user ID in request"
-        body:
-          text/plain:
-            example: "Bad request, e.g. invalid user UUID"
-      500:
-        description: "Internal server error, e.g. due to misconfiguration"
-        body:
-          text/plain:
-            example: "Internal server error, contact administrator"
+            type: synchronizationRequest
+        responses:
```

Comment:
Should there also be 422 code in case of validation failure?

Reply:
added